### PR TITLE
Require PHP 8.2

### PR DESCRIPTION
**Pull request by OskarStark**
I will execute rector for 8.2 in a follow up PR to make this PR less noisy

### CODE REVIEW

**Comment by OskarStark** [comment]
Ready to merge after release

**Comment by OskarStark** [comment]
CleanShot 2024-08-29 at 10 23 09@2x

You need to make all 8.2 now required instead of the 7.4 CI status names

File: rector.php [hunk 1]
```diff
@@ -2,13 +2,22 @@
...
 return RectorConfig::configure()
     ->withPaths([
-        __DIR__ . '/examples',
-        __DIR__ . '/src',
-        __DIR__ . '/tests',
+        __DIR__.'/examples',
+        __DIR__.'/src',
+        __DIR__.'/tests',
     ])
     ->withPhp74Sets()
```
Comment (slunak) [inline rector.php:22]: Should here still be `->withPhp74Sets()`?

Comment (OskarStark) [reply rector.php:22]: Yes because I changed and executed this in the follow up PR

**Review by slunak — APPROVED** [review]
Thank you!

**Pull request merged**